Makefile

```diff
@@ -65,10 +65,11 @@ endif
 
 #### Tools ####
 
-ifeq ($(shell type mips-linux-gnu-ld >/dev/null 2>/dev/null; echo $$?), 0)
-  MIPS_BINUTILS_PREFIX := mips-linux-gnu-
-else
-  $(error Please install or build mips-linux-gnu)
+MIPS_BINUTILS_PREFIX := mips-linux-gnu-
```

Comment:
Could you bring this to the top in the Defaults section with a comment? Also from what I remember I think we need to use `?=` to allow overriding in `.make_options`, though I could be wrong.

Reply:
Yeah `:=`/`=` will replace any previous value, `?=` doesn't